### PR TITLE
sem (schema-evolution-manager) 0.9.40 (new formula)

### PR DESCRIPTION
**Pull request by gheine**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
In the `install.sh` script, the line:
`mkdir -p "$PREFIX"/{bin,src}`
should be:
`mkdir -p "$PREFIX"/{bin, lib}`

**Comment by gheine** [comment]
@fxcoudert thx a mill. Fixed.

**Comment by fxcoudert** [comment]
```
  * Non-libraries were installed to "/usr/local/opt/sem/lib"
    Installing non-libraries to "lib" is discouraged.
    The offending files are:
      /usr/local/opt/sem/lib/schema-evolution-manager.rb
  * Non-executables were installed to "/usr/local/opt/sem/bin"
    The offending files are:
      /usr/local/opt/sem/bin/sem-config
```

File: Formula/sem.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 "cf5aab3f2fe81e3daca400811f46a11d4ebd25d7325eca92a83dbe0494e25d5a"
+
+  bottle :unneeded
+
+  def install
+    system "./install.sh", prefix
+  end
+
+  test do
+    assert_equal "0.9.40", shell_output("#{bin}/sem-info version").chomp
```
Comment (fxcoudert) [inline Formula/sem.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/sem.rb (hunk 1 above)
Comment (gheine) [reply Formula/sem.rb]: @fxcoudert added a better test

**Comment by gheine** [comment]
@fxcoudert can this be merged now?

**Comment by commitay** [comment]
https://github.com/Homebrew/homebrew-core/pull/28816

**Pull request closed without merging**